### PR TITLE
Fix #7157

### LINKED ISSUE #7157: Stereo Annotation Appears Incorrect
**Describe the bug**
When rendering the molecule `C[S@@](=O)C1=CC=CC=C1`, MarvinJS and Biovia ChemDraw both render `(R)` (left):
image

RDKit renders the same molecule with `(S)` (right):
image

**To Reproduce**
Render the smiles string `C[S@@](=O)C1=CC=CC=C1` with RDKit.

**Expected behavior**
I expect to see an `(R)` instead of an `(S)` chirality indicator.

**Screenshots**

**Configuration (please complete the following information):**
 - RDKit version: [@rdkit/rdkit 2023.9.2-1.0.0](https://www.npmjs.com/package/@rdkit/rdkit/v/2023.9.2-1.0.0)
 - OS: MacOS (Node 20)
 - Python version (if relevant): N/A
 - Are you using conda? No
 - If you are using conda, which channel did you install the rdkit from? N/A
 - If you are not using conda: how did you install the RDKit? npm

**Additional context**

### PR DESCRIPTION
**Pull request by ptosco**
- fix #7157
- add the `assignCIPLabels` flag to MinimalLib's `mol_from_input()`
- added cffi and JS tests
- refactored `process_details()`, `process_mol_details()` and `process_rxn_details()` to take a `struct` reference as parameter rather than a long list of parameters
- this also allowed to get rid of one of the ugly `GET_JSON` macros
- exposed `panelWidth`, `panelHeight `and `noFreetype` (the latter is useful for testing)
- made `width` and `height` to default to -1 in `mol_to_svg` since we now support flexicanvas

### CODE REVIEW

**Review by greglandrum — APPROVED** [review]
LGTM

I can't really comment on the changes in the JS API, but the rest looks fine.

**Comment by ptosco** [comment]
@greglandrum Are you waiting for input from my side before merging?

**Comment by greglandrum** [comment]
> @greglandrum Are you waiting for input from my side before merging?

not exactly... I was either waiting for exactly the right time or I just forgot to do it
I will leave it to you decide which is true. ;-)

**Comment by ptosco** [comment]
@greglandrum I think you can figure out from the polite neutrality of my question what is my guess :-)

**Pull request merged**
